bundle/regal/rules/style/detached-metadata/detached_metadata.rego

```diff
@@ -29,3 +28,16 @@ _annotation_at_row(row) := annotation if {
 
 	util.to_location_object(annotation.location).row == row
 }
+
+_allow_detached(last_row, i, blocks, lines) if {
```

Comment:
I understand this now, but a comment might be helpful in a few months time! 😅

Reply:
Sure! :) I'll add one 👍